SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -6,5 +6,6 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
 | OpenMamba | [Openmamba GPLv2 License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt) | bash-completion |
-| TreasureData | [TreasureData ASL 2.0 License](http://www.apache.org/licenses/LICENSE-2.0) | td-agent |
+| TreasureData | [TreasureData ASL 2.0 License](http://www.apache.org/licenses/LICENSE-2.0) |td-agent |
```

Comment:
The two ASL licenses are the same. I suggest dropping the prefix, which might suggest that the license is somehow different from the original.

Reply:
changed to show they are the same license